index.html

```diff
@@ -2933,6 +2943,10 @@ <h2 class="code-line" data-line-start=9 data-line-end=10><a id="Companies_that_r
 			</tbody>
 		</table>
 		<p></p> 
+		<h2 class="code-line" data-line-start=2 data-line-end=3><a id="Notes_3"></a>Do not give up!</h2>
+		<p>Sure, Software Engineers are the most wanted IT specialists but from experience, companies are much more likely to seek candidates for QAs, PMs, Designers and other vacancies.</p>
+		<p>Keep learning English and keep trying!</p>
```

Comment:
I would also add there something like "Keep learning English and technical skills. Keep trying!"